pkg/util/templater/templater.go

```diff
@@ -51,11 +50,10 @@ func (r *Templater) Render(content string, context map[string]interface{}, snipp
 
 	// @step: add the snippits into the mix
 	for filename, snippet := range snippets {
-		name := filepath.Base(filename)
```

Comment:
This bit resolves overloading snippets. Previously you could load two snippet files overwriting the same var and they would be applied in an inconsistent order (due to no ordering in the map), causing you to potentially have a different cluster spec generated on each run.

Thanks! :)